src/sentry/models/eventattachment.py

```diff
@@ -44,12 +44,17 @@ class Meta:
 
     @cached_property
     def mimetype(self):
-        rv = self.file.headers.get("Content-Type")
+        from sentry.models import File
+
+        file = File.objects.get(id=self.file_id)
```

Comment:
I wonder if we should fake this foreign key by doing something like, `_cached_file` like we/Django does internally to bind the object.

My concern is we might be introducing a bunch of duplicate queries to fetch the File object instead of fetching it once, and binding it, then passing alone. 

We could mimic this behavior with a property or something to mask if. Maybe just a cached_property for `.file`?

Reply:
I was thinking about something like this... It shouldn't matter for this specific property since it's already cached, but it might make things a bit more usable.